### DISCUSSION TITLE
Nodes appear offline in Google Home app

### DISCUSSION THREAD

**Discussion by drbeat** [andrei-tatar/node-red-contrib-smartnora#120] [Q&A]
So I seem to be stuck somewhere..
my setup:
Node-RED in Docker with network set to host
logs arent showing issues
`> node-red-docker@2.1.3 start /usr/src/node-red,
> node $NODE_OPTIONS node_modules/node-red/red.js $FLOWS "--userDir" "/data",
23 Nov 08:29:04 - [info] ,
Welcome to Node-RED,
===================,
23 Nov 08:29:04 - [info] Node-RED version: v2.1.3,
23 Nov 08:29:04 - [info] Node.js  version: v14.18.1,
23 Nov 08:29:04 - [info] Linux 5.10.63-v7l+ arm LE,
23 Nov 08:29:04 - [info] Loading palette nodes,
23 Nov 08:29:07 - [info] Settings file  : /data/settings.js,
23 Nov 08:29:07 - [info] Context store  : 'default' [module=memory],
23 Nov 08:29:07 - [info] User directory : /data,
23 Nov 08:29:07 - [warn] Projects disabled : editorTheme.projects.enabled=false,
23 Nov 08:29:07 - [info] Flows file     : /data/flows.json,
23 Nov 08:29:07 - [info] Server now running at http://127.0.0.1:1880/,
23 Nov 08:29:07 - [warn] ,
---------------------------------------------------------------------,
Your flow credentials file is encrypted using a system-generated key.,
If the system-generated key is lost for any reason, your credentials,
file will not be recoverable, you will have to delete it and re-enter,
your credentials.,
You should set your own key using the 'credentialSecret' option in,
your settings file. Node-RED will then re-encrypt your credentials,
file using your chosen key the next time you deploy a change.,
---------------------------------------------------------------------,
23 Nov 08:29:07 - [info] Starting flows,
23 Nov 08:29:08 - [info] Started flows,
23 Nov 08:29:08 - [info] [tasmota-mqtt-broker:mosquitto] Connected to broker: mqtt://localhost:1883,
23 Nov 08:29:08 - [info] nora: authenticated, uid: Hn3i6PEY2tdmjbx7JB6bVWyjPFy1,
23 Nov 08:29:08 - [info] nora: Zuhause - disconnected,
23 Nov 08:29:08 - [info] nora: Zuhause - connected,
23 Nov 08:29:15 - [info] nora: Zuhause - synced 3 device(s),
23 Nov 08:35:30 - [info] Stopping flows,
23 Nov 08:35:30 - [info] [tasmota-mqtt-broker:mosquitto] Disconnected from broker: mqtt://localhost:1883,
23 Nov 08:35:30 - [info] Stopped flows,
23 Nov 08:35:30 - [info] Starting flows,
23 Nov 08:35:30 - [info] nora: Zuhause - connected,
23 Nov 08:35:30 - [info] Started flows,
23 Nov 08:35:30 - [info] [tasmota-mqtt-broker:mosquitto] Connected to broker: mqtt://localhost:1883,
23 Nov 08:35:37 - [info] nora: Zuhause - synced 3 device(s),
23 Nov 08:45:50 - [info] Stopping flows,
23 Nov 08:45:50 - [info] [tasmota-mqtt-broker:mosquitto] Disconnected from broker: mqtt://localhost:1883,
23 Nov 08:45:50 - [info] Stopped flows,
23 Nov 08:45:50 - [info] Starting flows,
23 Nov 08:45:50 - [info] nora: Zuhause - connected,
23 Nov 08:45:50 - [info] Started flows,
23 Nov 08:45:50 - [info] [tasmota-mqtt-broker:mosquitto] Connected to broker: mqtt://localhost:1883,
23 Nov 08:45:54 - [info] nora: Zuhause - synced 3 device(s)`

I tried connecting my Tasmota devices -> outlet which doesnt seem to be working since it has a glow in the app and it show "offline"
same for the outlet example and any other node

any ideas what I could change?

**Comment by andrei-tatar** [comment]
@drbeat what version of the package are you using?
What do you see in the groups when going to https://smart-nora.eu/my-nora ?

**Comment by drbeat** [comment]
Hi Andrei,
I am using v1.5.1 of smartnora.
In the group are 2 devices listed (as I have configured)
`Group: Zuhause - 2 device(s)
Last sync 11/23/21, 4:28 PM
Last heartbeat 1 hour(s) ago
by node-red-contrib-smartnora/1.5.1`

My Node-RED has access to the internet as other services seem to be working fine

**Comment by andrei-tatar** [comment]
What device are you using? How is it connected to the internet? Are you using any sort of VPN/proxy? It seems like it can't maintain an active connection to the firebase realtime databse.

**Comment by drbeat** [comment]
It is running on a RPi 4 with LAN directly connected to my router.
Not using a VPN/Proxy. It is strange as other services work with my node-red setup.

I know that my Router does not allow incoming connections but is the case here?

**Reply by andrei-tatar** [reply]
> I know that my Router does not allow incoming connections but is the case here?

No, shouldn't be the case.

Can you check if you correctly configured username/password in the config node? I think someone else mentioned the nora configuration weirdly vanished: https://github.com/andrei-tatar/node-red-contrib-smartnora/issues/117 (seems to be running exactly the same node-red version as you: 2.1.3)

Maybe it's a bug in a later node-red version...

**Reply by drbeat** [reply]
I looked into it. Was correctly configured.
Nevermind I gave up, seems like it doesnt want to work. :)
Thanks anyways
